dxp/liferay_global_event_handler.md

```diff
@@ -0,0 +1,213 @@
+# Liferay’s Global Event Handler
+
+Liferay’s event handler helps the developers for creating/listening/firing/removing custom events that can be listened in whole DXP which is rely on the YUI3 event target implementation. See the implementation [here](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120).
+
+### First Considerations
+
+First, just consider using the global event handler if necessary. Try to use the default event listener/[callback pattern(if React)](https://reactjs.org/docs/lifting-state-up.html#lifting-state-up) when working. Just use the global event handler when trying to communicate between widgets or not accessible contexts(e.g iframes).
+
+### List of Available Functions
+
+Here is a list of available functions from Liferay’s global variable:
```

Comment:
If you wanted to do the "less is more" thing, heading + sentence could be reduced to just heading:

```
### Functions
```

Reply:
It looks more simpler, yeh? I'll update :)

Reply:
Updated!